lib/fabric/app/models/fabric/invoice_item.rb

```diff
@@ -6,22 +6,54 @@ class InvoiceItem
 
     belongs_to :customer, class_name: 'Fabric::Customer',
       primary_key: :stripe_id, touch: true
+    belongs_to :invoice, class_name: 'Fabric::Invoice', primary_key: :stripe_id
+    belongs_to :subscription, class_name: 'Fabric::Subscription',
+      primary_key: :stripe_id
 
     field :stripe_id, type: String
     field :amount, type: Integer
-    field :invoice, type: String
     field :currency, type: String
+    field :date, type: Time
+    field :description, type: String
+    field :discountable, type: Boolean
+    field :discounts, type: Array
+    field :livemode, type: Boolean
+    field :metadata, type: Hash
+    field :period, type: Hash
+    field :price, type: Hash
+    field :proration, type: Boolean
+    field :quantity, type: Integer
+    field :subscription_item, type: String
```

Comment:
I'm confused about this field. I notice you didn't sync it below, and I was going to add it, but then I figured I'd check and see if it showed up on an InvoiceItem retrieved from the Stripe API, since they list it in the field definitions but they don't include it in their example object ([on the right in the docs](https://stripe.com/docs/api/invoiceitems/object)). I retrieved an invoice created today and it seems that subscription_item isn't present, and if you try to call `invoice_item.subscription_item`, it raises an error.

Were you aware of this? I'm not sure if we should include the field but not sync it, or what we should do here.

Reply:
I forgot to remove the field after I ran into the issue of the method not existing on invoice items. I tried updating the gem, but that didn’t resolve it. Figured it was just simpler to leave out for now, but the field should be removed, just accidentally forgot.

Reply:
@jclusso This has come up before. I wish I knew why this happens. I really don't like it.